src/style/Header.css

```diff
@@ -21,8 +21,12 @@ header{
     height: 100%;
     width: 100%;
     & button{
+      color: #000;
       width: 100%;
       height: 100%;
+      &:active{
+        color: #aaa;
+      }
     }
   }
 }
```

Comment:
ditto

Reply:
Not quite understand here tho

Reply:
<img width="422" alt="Use_icons_at_Header__Footer__this_fixes__10_by_AsukaCHikaru_·_Pull_Request__13_·_project-yori_yori-view" src="https://user-images.githubusercontent.com/644741/60453443-26a10a80-9c6c-11e9-80e6-017a810f73b7.png">
This needs an extra line feed